src/winconpty/winconpty.h

```diff
@@ -19,6 +19,9 @@ typedef struct _PseudoConsole
 //      the signal pipe.
 #define PTY_SIGNAL_RESIZE_WINDOW (8u)
 
+// The other flag (PSEUDOCONSOLE_INHERIT_CURSOR) is actually defined in consoleapi.h in the OS repo
+#define PSEUDOCONSOLE_RESIZE_QUIRK (2u)

```

Comment:
This is in a misleading place. This makes it look like it's a signal and not a startup flag.